app/containers/ExplorePage.tsx

```diff
@@ -90,19 +156,36 @@ export default function ExplorePage() {
     certaintyRange: [0, 1]
   });
   const [data, setData] = useState<undefined | ObservationsData>();
+  const [inspectedObservations, setInspectedObservations] = useState<
```

Comment:
i suppose this styling is enforced by some rules, but makes me sad to see this broken into 3 lines like this... Doesn't really increase readability. if possible try to fit this in one line? maybe we need to increase line length limit? or use some shorter name for this part of state

Reply:
Done; I hated this too :)